### PR TITLE
bpo-27500: Fix sttaic ver of getaddrinfo to resolve IPv6 addresses

### PR DESCRIPTION
**Pull request by 1st1**
https://bugs.python.org/issue27500

### CODE REVIEW

File: Lib/test/test_asyncio/test_base_events.py [hunk 1]
```diff
@@ -1077,6 +1077,26 @@ def test_create_server_stream_bittype(self):
             srv.close()
             self.loop.run_until_complete(srv.wait_closed())
 
+    @unittest.skipUnless(hasattr(socket, 'AF_INET6'), 'no IPv6 support')
```
Comment (vstinner) [inline Lib/test/test_asyncio/test_base_events.py:1080]: You might use support.IPV6_ENABLED

**Review by vstinner — APPROVED** [review]
LGTM.

Be carefully of the typo in the commit message: "bpo-27500: Fix sttaic ver of getaddrinfo..."

**Comment by 1st1** [comment]
Thanks for a review, Victor!

**Review by asvetlov — COMMENTED** [review]
I'm late but lgtm

**Pull request merged**